cmd/podman/containers_prune.go

```diff
@@ -40,7 +40,7 @@ func init() {
 	pruneContainersCommand.SetHelpTemplate(HelpTemplate())
 	pruneContainersCommand.SetUsageTemplate(UsageTemplate())
 	flags := pruneContainersCommand.Flags()
-	flags.BoolVarP(&pruneContainersCommand.Force, "force", "f", false, "Force removal of a running container.  The default is false")
+	flags.BoolVarP(&pruneContainersCommand.Force, "force", "f", false, "Skip interactive prompt for container removal.")
```

Comment:
Can we remove the trailing dot?

Reply:
I'd rather add them to all others, but yeah, to be consistent go with @vrothberg